weather.js

```diff
@@ -0,0 +1,36 @@
+var city= "Beograd"
+
+function myFunction() {
+     city = document.getElementById("myText").value;
+    console.log(city);
```

Comment:
Always remove any console.logs unless they have a good reason to stay in your commit

Reply:
Understood